testsuite/forge-cli/src/main.rs

```diff
@@ -438,6 +439,7 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             state_sync_perf_fullnodes_execute_transactions(config)
         }
         "state_sync_perf_validators" => state_sync_perf_validators(config),
+        "validators_join_and_leave" => validators_join_and_leave(config),
```

Comment:
You also need to add a yaml file like https://github.com/aptos-labs/aptos-core/blob/eef8707b3cad07523a02ba207cd607380b5d2d94/.github/workflows/continuous-e2e-account-creation-test.yaml#L12 for your test to be running in continous mode